app/(gcforms)/[locale]/(form administration)/form-builder/[id]/responses/[[...statusFilter]]/components/ActionsPanel.tsx

```diff
@@ -4,7 +4,7 @@ import React from "react";
 export const ActionsPanel = ({ children }: { children: React.ReactNode }) => {
   return (
     <section className="fixed bottom-0 left-0 z-100 h-32 w-full border-t-2 border-black bg-white py-8">
-      <div className="mx-4 laptop:mx-24">
+      <div className="mx-20 tablet:mx-24">
```

Comment:
Moved button over as it gets covered a bit by the left nav when on a small screen.